building-with-ocean/asset-hosting.md

```diff
@@ -17,134 +17,136 @@ On Ocean Marketplace, a publisher must provide the link to the asset during publ
 
 Publishers can choose any hosting service of their choice. The below section explains how to use commonly used hosting services with Ocean Marketplace.
```

Comment:
Here, we need to give people a recommendation not to use Google Drive as it is not a reliable data storage solution that integrates well with Ocean

Reply:
Should I also mention here some issues reported in the past?

Reply:
We can mention that we encountered multiple issues and if you want your asset to be purchased and consumed, please avoid using GDrive